firefly_client/firefly_client.py

```diff
@@ -153,6 +158,32 @@ def __init__(self, url=_my_url, channel=None, html_file=_my_html_file,
         self.session = requests.Session()
         self.connect()
 
+        channel_matches = channel == os.environ['fireflyChannelLab']
+        url_matches = url == _my_url
+
+        if ('fireflyLabExtension' in os.environ) and use_lab_env \
+                and (not channel_matches or not url_matches):
+                print('Cannot use Jupyter lab environment: channel or url differ from default lab environment setup.')
+                if not url_matches:
+                    print('>>> Default URL: %s, Passed URL, %s' % (_my_url, url))
+                if not channel_matches:
+                    print('>>> Default channel: %s, Passed channel, %s' % (os.environ['fireflyChannelLab'], channel))
+                print('To disable this message pass False to use_lab_env')
+                use_lab_env = False
+
```

Comment:
Would it make sense to set `start_lab_tab = False` at this point?

Reply:
ok